src/config/queryClient.js

```diff
@@ -0,0 +1,28 @@
+import { configureQueryClient } from '@graasp/query-client';
+import { API_HOST } from './constants';
+import notifier from './notifier';
+
+const {
+  queryClient,
+  QueryClientProvider,
+  hooks,
+  useMutation,
+  ReactQueryDevtools,
+  API_ROUTES,
+} = configureQueryClient({
+  API_HOST,
+  enableWebsocket: true,
```

Comment:
Should this use the environment variable instead?

Reply:
I guess we can disable it actually, no need for websockets here, right?